File: java/src/main/java/io/ticktok/example/ManualTick.java
package io.ticktok.example;

import io.ticktok.client.Ticktok;

import static io.ticktok.client.Ticktok.options;

public class ManualTick {

    public static void main(String[] args) {
        final Ticktok ticktok = new Ticktok(options().domain(args[0]).token(args[1]));
        ticktok.schedule("non-scheduled-tick", "@never", () -> System.out.println("tick"));
        ticktok.tick("non-scheduled-tick", "@never");
    }
}

File: clojure/examples/src/examples/manual_tick.clj
(ns examples.manual-tick
  (:require [ticktok.core :as tk])
  (:gen-class))

(defn -main
  ""
  [& args]
  (let [[host token _] args
        ticktok (tk/ticktok :start {:host host
                                    :token token})]

    (ticktok :schedule {:name "non-scheduled-tick"
                        :schedule "@never"
                        :callback #(println "tick")})

    (ticktok :tick {:name "non-scheduled-tick"
                   :schedule "@never"})))
